cfpb/cfgov-refresh
# Update wellbeing app for final testing functionality and content

A few final fixes before tomorrow's usability testing of the Financial Well-Being Scale app.

## Additions

- Missing content

## Changes

- Fix histogram selection on primary results page
- Improve a few bits of styling
- Fix how Django processes the templates for this app

## Testing

1. Pull branch
2. Run server
3. Visit http://localhost:8000/consumer-tools/financial-well-being/results/?86
4. Play with the histogram selector
5. Visit http://localhost:8000/consumer-tools/financial-well-being/about/
6. See all the shiny new content

## Checklist
